examples/IRServer/IRServer.ino

```diff
@@ -132,6 +132,8 @@ void setup(void) {
   if (mdns.begin(HOSTNAME)) {
 #endif  // ESP8266
     Serial.println("MDNS responder started");
+    // Announce esp tcp service on port 8080
+    mdns.addService("esp", "tcp", 8080);
```

Comment:
Shouldn't this be port `80` rather then `8080`?
See lines 51 to 56.

Also "http" rather then "esp".

Reply:
I must have been looking at an example of defining a custom non-http service. But the browsers worked any way. I will fix and verify it works with Firefox and Chromium.

Reply:
Thanks. Looking forward to the results of your testing.

Reply:
In all cases, I tested using Ubuntu 18.04 running the latest versions of FireFox and Chromium. I let the package manager update the browsers when needed (which seems to be fairly often).

IRServer and IRMQTTServer on ESP8266 (d1_mini) work fine.

On ESP32 (Espressif dev board), IRServer and IRMQTTServer work fine.

Thanks for your work on this library. I can now watch TV without juggling 3 remote controls!

Reply:
Thanks for confirming. And thanks even more for fixing it.